fix: broken README link to Pi imager

rogerlz opened the pull request:
This PR resolves issue #230 by directing users to the updated MainsailOS documentation. It also updates the repository links for Klipper and Mainsail after their relocation to new organizations.

meteyou commented:
Thx for the PR! i think the link above `You can find detailed instructions in our [documentation](https://docs.mainsail.xyz/setup/mainsail-os).` should also be changed. can you do it and add a commit to this PR?

rogerlz commented:
> Thx for the PR! i think the link above `You can find detailed instructions in our [documentation](https://docs.mainsail.xyz/setup/mainsail-os).` should also be changed. can you do it and add a commit to this PR?

yes of course 👍 
fixed and squashed

meteyou commented:
Thx for your PR!
